repos/spack_repo/builtin/packages/py_pandas/package.py

```diff
@@ -91,11 +92,8 @@ class PyPandas(PythonPackage):
 
     with default_args(type=("build", "run")):
         # Based on PyPI wheel availability
-        depends_on("python@3.10:", when="@2.2:")
```

Comment:
This line is incorrect, I don't know where this came from. Other min versions are deprecated anyway.